.circleci/config.yml

```diff
@@ -182,7 +182,7 @@ jobs:
           name: Install Homebrew packages
           command: |
             brew update
-            brew install automake libtool cmake pkg-config wget meson
+            brew install automake libtool cmake pkg-config wget meson python@3.11
```

Comment:
Just `python` should be sufficient as Homebrew doesn't support python2: https://docs.brew.sh/Homebrew-and-Python